The Makefile doesn't provide a way to install into a custom directory

--- old/Makefile
+++ new/Makefile
@@ -26,7 +26,7 @@
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #
 
-prefix=/usr/local
+prefix=$(DESTDIR)/usr
 
 datarootdir=$(prefix)/share
 docdir=$(datarootdir)/doc/gitflow
